README.md

```diff
@@ -20,7 +20,7 @@
   </a>
 </p>
 
-Amundsen is a metadata driven application for improving the productivity of data analysts, data scientists and engineers when interacting with data. It does that today by indexing data resources (tables, dashboards, streams, etc.) and powering a page-rank style search based on usage patterns (e.g. highly queried tables show up earlier than less queried tables). Think of it as Google search for data. The project is named after Norwegian explorer [Roald Amundsen](https://en.wikipedia.org/wiki/Roald_Amundsen), the first person to discover the South Pole.
+Amundsen is a data discovery and metadata engine for improving the productivity of data analysts, data scientists and engineers when interacting with data. It does that today by indexing data resources (tables, dashboards, streams, etc.) and powering a page-rank style search based on usage patterns (e.g. highly queried tables show up earlier than less queried tables). Think of it as Google search for data. The project is named after Norwegian explorer [Roald Amundsen](https://en.wikipedia.org/wiki/Roald_Amundsen), the first person to discover the South Pole.
```

Comment:
`a Data Discovery And Metadata Platform` which matches https://eng.lyft.com/open-sourcing-amundsen-a-data-discovery-and-metadata-platform-2282bb436234

Reply:
haha, this one says `engine` https://eng.lyft.com/amundsen-lyfts-data-discovery-metadata-engine-62d27254fbb9

Reply:
🤦

Reply:
@feng-tao, how strongly do you feel about engine vs. platform?

Reply:
lgtm, I think engine is fine as well, sorry, in a meeting

Reply:
ok, then. Thanks in advance for merging!